bytes.md

```diff
@@ -0,0 +1,50 @@
+---
+title: BYTES
+toc: true
+---
+
+## Description
+
+The `BYTES` [data type](data-types.html) stores binary strings of variable, unlimited length.
```

Comment:
While the length isn't currently limited, it is bold to claim `unlimited length`. It is likely a bad idea to store a value that is larger than the range size (64 MB). I'm not exactly sure how to phrase the length recommendation.